src/sui-link-item/index.jsx

```diff
@@ -51,7 +53,8 @@ export default class LinkItem extends Component {
           target={target}
           title={title}
           to={url}
-          onClick={handleClick}>
+          onClick={handleClick}
+          rel={rel}>
```

Comment:
Can you check if that's working? I mean, I honestly don't know if the Link React Router is going to add this attribute.